runner/operator/argos_operator/v1_1_2/bin/make_sample.py

```diff
@@ -265,12 +265,15 @@ def _set_pu(self):
         pu = list()
         for f in self.r1:
             metadata = get_file(f).metadata
+            flowcell_id = "MT_FCID"
```

Comment:
I think in these cases you can always do `metadata('flowCellId', "MT_FCID")` if metadata['flowCellId'] = None. But this is good also.

Reply:
Good point! I'll make a note of it for future releases.